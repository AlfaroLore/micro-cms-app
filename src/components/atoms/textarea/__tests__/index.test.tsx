import { render } from '@testing-library/react';
import TextArea from '..';

describe('TextArea', () => {
  it('Renders TextArea', () => {
    const component = render(<TextArea />);

    expect(component).toBeTruthy();
    expect(component.asFragment()).toMatchSnapshot();
  });
});
