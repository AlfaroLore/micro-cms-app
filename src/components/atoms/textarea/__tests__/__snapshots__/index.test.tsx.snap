// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TextArea > Renders TextArea 1`] = `
<DocumentFragment>
  <textarea
    class="sc-bgqQcB hVQiYn"
  />
</DocumentFragment>
`;
